README.md

```diff
@@ -4,14 +4,13 @@ MUSIC BLOCKS
 Music Blocks is a collection of manipulative tools for exploring
 fundamental musical concepts in an integrative and fun way.
 
-<img
```

Comment:
Sir, I am using windows 7.
Below is the output with chrome and windows 7
![bug2](https://cloud.githubusercontent.com/assets/15171633/24089043/b4136dbe-0d57-11e7-8e09-6b632e768cf6.PNG)

the readme with mozilla in ubuntu
![mozillabug](https://cloud.githubusercontent.com/assets/15171633/24089071/1206c312-0d58-11e7-99fc-4c31ffdaacd3.png)

I also checked it on my android phone and chrome
![screenshot_20170320-103155](https://cloud.githubusercontent.com/assets/15171633/24089115/b1cf16ec-0d58-11e7-81bb-6a5872516488.png)